gen/dcos-config.yaml

```diff
@@ -489,30 +489,34 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
-  - path: /etc_master/3dt_config_open.json
+  - path: /etc/dcos-3dt.env
+    content: |
+      DCOS_3DT_CONFIG_PATH=/opt/mesosphere/etc/dcos-3dt-config.json
+  - path: /etc_master/dcos-3dt-config.json
     content: |
       {
         "role": "master",
         "pull": true,
         "diagnostics-bundle-dir": "/var/lib/dcos/3dt/diag-bundles",
-        "endpoint-config": "/opt/mesosphere/etc/endpoints_config_open.json",
+        "endpoint-config": "/opt/mesosphere/etc/dcos-3dt-endpoint-config.json",
+        "no-unix-socket": true,
```

Comment:
Or you could just default "unix-socket" to `true`, and then this line overrides it to false until you're ready to remove it.
But that's just my personal pet peeve about positive-case variable names. Call me an optimist. We can still ship like this.